pkg/util/http.go

```diff
@@ -0,0 +1,6 @@
+package util
+
+const (
+	ContentTypeHeaderKey    = "Accept"
```

Comment:
call this `AcceptHeaderKey`.  There is another standard header called "ContentType"

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type